code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -23,6 +23,31 @@
 	use_to_pickup = 1
 	slot_flags = SLOT_BELT
 
+/obj/item/storage/bag/trash/proc/update_weight()
```

Comment:
You are all the code next to ```storage/bag``` and not ```storage/bag/trash``` which is next to it.

Reply:
Resolved, sorry for the delay!